### PR TITLE
[12.0] [MIG] sale_stock_picking_blocking

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of sale_stock_picking_blocking to version 12.0

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test on runbot. It works as expected.

File: sale_stock_picking_blocking/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    default_delivery_block = fields.Many2one(
+        comodel_name='sale.delivery.block.reason',
+        string='Default Delivery Block Reason',
+        help="Set a reason to block by default the deliveries in this "
+             "customer sales orders.")
+
+    @api.model
+    def _commercial_fields(self):
+        commercial_fields = super(Partner, self)._commercial_fields()
+        commercial_fields.append('default_delivery_block')
```
Comment (nikul-serpentcs) [inline sale_stock_picking_blocking/models/res_partner.py]: @bjeficent  `return commercial_fields.append('default_delivery_block')`

File: sale_stock_picking_blocking/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<!--Copyright 2019 Eficent Business and IT Consulting Services S.L.-->
+    <!--(http://www.eficent.com)-->
+<!--License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).-->
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group/field[@name='user_id']" position="after">
```
Comment (nikul-serpentcs) [inline sale_stock_picking_blocking/views/res_partner_view.xml]: @bjeficent Direct use `field`

File: sale_stock_picking_blocking/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    def onchange_partner_id(self):
+        """Add the 'Default Delivery Block Reason' if set in the partner."""
+        res = super(SaleOrder, self).onchange_partner_id()
+        for so in self:
+            so.delivery_block_id = so.partner_id.default_delivery_block or \
+                False
+        return res
+
+    @api.multi
+    def action_remove_delivery_block(self):
```
Comment (nikul-serpentcs) [inline sale_stock_picking_blocking/models/sale_order.py:38]: return ??
Put just return True

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs do you consider your comments are attended?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Pull request merged**